src/routes/redux/detailsSlice.js

```diff
@@ -0,0 +1,68 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const detailsSlice = createSlice({
+  name: "details",
+  initialState: {
+    value: {
+      name: "",
+      lastName: "",
+      phone: "",
+      address: "",
+      shirtSize: "",
+      pantsSize: "",
+      budget: "",
+      neverWear: [],
+      alwaysWear: [],
+    },
+  },
+  reducers: {
+    updateDetails: (state, action) => {
+      state.value = {
+        ...state.value,
+        [action.payload.fieldName]: action.payload.value,
+      };
+    },
+    updateNeverWear: (state, action) => {
+      let value = action.payload.value;
+      state.value.neverWear = [...state.value.neverWear, value];
+    },
+    updateAlwaysWear: (state, action) => {
+      let value = action.payload.value;
+      state.value.alwaysWear = [...state.value.alwaysWear, value];
+    },
+    updateNeverWearDelete: (state, action) => {
+      let value = action.payload.value;
+      let newStateNeverWear = [...state.value.neverWear];
+
+      for (let i = 0; i < newStateNeverWear.length; i++) {
+        if (newStateNeverWear[i] === value) {
+          newStateNeverWear.splice(i, 1);
+          state.value.neverWear = [...newStateNeverWear];
+        }
+      }
```

Comment:
Kol Hakavod for making it work!
you can clean up the code a little. try looking for cleaner ways to remove an item from an array
https://love2dev.com/blog/javascript-remove-from-array/
https://stackoverflow.com/questions/5767325/how-can-i-remove-a-specific-item-from-an-array
https://www.geeksforgeeks.org/remove-elements-from-a-javascript-array/